pennylane/labs/vibrational/pes_generator.py

```diff
@@ -15,30 +15,27 @@
 per normal modes on a grid."""
 
 import itertools
-import subprocess
+from pathlib import Path
 
 import numpy as np
 
 import pennylane as qml
 from pennylane.data.base._lazy_modules import h5py
-
-from .localize_modes import localize_normal_modes
-from .vibrational_class import (
-    VibrationalPES,
+from pennylane.qchem import VibrationalPES, localize_normal_modes, optimize_geometry
+from pennylane.qchem.vibrational.vibrational_class import (
     _get_dipole,
     _harmonic_analysis,
     _single_point,
-    optimize_geometry,
 )
 
-# pylint: disable=too-many-arguments, too-many-function-args, c-extension-no-member
+# pylint: disable=too-many-arguments, too-many-function-args, c-extension-no-member, too-many-branches
 # pylint: disable= import-outside-toplevel, too-many-positional-arguments, dangerous-default-value
```

Comment:
Can we remove some of these checks?  
We really shouldn't be setting `dangerous-default-value` anywhere in the codebase, as it is an indication that the implementation can/will produce wrong results under certain use-cases.

Reply:
Removed a couple of them. too-many-branches arises because we are dealing with some large objects (4 dimensional and 6 dimensional arrays) and need to loop over these. The too-many-args ones are for private functions.

Reply:
Thanks @ddhawan11 
Sure, that's fine. I was mostly concerned with the dangerous default value scenario. For too many args and such, we can always keep those for now until as they shouldn't be worrisome .